open-api.yaml

```diff
@@ -709,28 +728,84 @@ components:
     taskUid:
       name: taskUid
       in: path
-      description: The task unique identifier
+      description: The task identifier
       required: true
+      schema:
+        type: integer
+    taskFilterUids:
+      name: uids
+      in: query
+      description: Permits to filter tasks by their uid. By default, when the `uids` query parameter is not set, all task uids are returned. It's possible to specify several uids by separathing them with the `,` character.
+      required: false
       schema:
         type: number
-    taskIndexUid:
-      name: indexUid
+    taskFilterIndexUids:
+      name: indexUids
+      in: query
+      description: Permits to filter tasks by their related index. By default, when `indexUids` query parameter is not set, the tasks of all the indexes are returned. It is possible to specify several indexes by separating them with the `,` character.
+      required: false
+      schema:
+        type: string
+    taskFilterTypes:
+      name: types
+      in: query
+      description: Permits to filter tasks by their related type. By default, when `types` query parameter is not set, all task types are returned. It's possible to specify several types by separating them with the `,` character.
+      required: false
+      schema:
+        type: string
+    taskFilterStatuses:
+      name: statuses
+      in: query
+      description: Permits to filter tasks by their status. By default, when `statuses` query parameter is not set, all task statuses are returned. It's possible to specify several statuses by separating them with the `,` character.
+      required: false
+      schema:
+        type: string
+    taskFilterBeforeEnqueuedAt:
+      name: beforeEnqueuedAt
+      in: query
+      description: Permits to filter tasks based on their enqueuedAt time. Matches tasks enqueued before the given date.
+      required: false
+      schema:
+        type: string
```

Comment:
I would like to have more information about the type of this filter, e.g: Can I send `2021-12-20` on it?

Reply:
Yes, mentioning that we support RFC3339 could at least be helpful; It's mentioned in the specification but it could be interesting to put that in the open API spec!